src/lib/input/input-container.ts

```diff
@@ -302,6 +336,12 @@ export class MdInputContainer implements AfterContentInit {
     this._mdInputChild._placeholderChange.subscribe(() => this._validatePlaceholders());
   }
 
+  ngAfterViewInit() {
+    // Avoid animations on load.
+    this._messageAnimationState = 'enter';
+    this._changeDetectorRef.detectChanges();
```

Comment:
@andrewseguin do you know if there's a better way to prevent animations on load?